blocks/library/quote/index.js

```diff
@@ -20,11 +20,21 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' )
 	},
 
+	controls: [ '1', '2' ].map( ( variation ) => ( {
+		icon: 'format-quote',
+		title: wp.i18n.sprintf( wp.i18n.__( 'Quote %d' ), variation ),
+		isActive: ( { style = '1' } ) => style === style,
+		onClick( attributes, setAttributes ) {
+			setAttributes( { style: variation } );
+		},
+		variation
```

Comment:
Looks like we've highlighted a problem that our `<Toolbar />` is tightly coupled to the idea of a heading level:

https://github.com/WordPress/gutenberg/blob/d068400/editor/components/toolbar/index.js#L24